deployment/keycloak/imports/realm.json

```diff
@@ -41,19 +42,22 @@
       "subGroups": []
     }
   ],
+  "defaultGroups": [
+    "/projectbuilder-default"
+  ],
   "clients": [
     {
       "id": "2f8e4455-dcbc-4535-8ec1-b1c3ced005ae",
       "clientId": "projectbuilder",
       "name": "projectbuilder",
-      "rootUrl": "http://localhost:8081",
-      "baseUrl": "http://localhost:8081",
+      "rootUrl": "",
+      "baseUrl": "",
       "surrogateAuthRequired": false,
       "enabled": true,
       "alwaysDisplayInConsole": false,
       "clientAuthenticatorType": "client-secret",
       "redirectUris": [
-        "http://localhost:8081/*"
+        "*"
```

Comment:
This value in production is dangerous because then the OIDC provider can be used by every application. localhost or the external URL must be specified here